catchup.sh

```diff
@@ -94,21 +94,29 @@ create_test() {
 create_sources() {
    echo "performing Exercise 8 (Create Package with sources)"
    cd create_sources
+   conan new hello/0.1 -t -s
+   conan create . user/testing
+   conan create . user/testing -s build_type=Debug
+}
+
+create_sources_scm() {
+   echo "performing Exercise 9 (Create Package with sources)"
```

Comment:
```suggestion
   echo "performing Exercise 9 (Create Package with sources SCM)"
```
I'd add somehting different to the description of the example to differentiate it from the other one with exports_sources